player/player.lua

```diff
@@ -59,6 +60,11 @@ end
 
 local player = setmetatable({}, {
     __index = function(_, k)
+
+        if k == 'position' or k == 'heading' or k == 'target_index' then
```

Comment:
This works fine for now, if it gets any more I would create a table keyed with names that should redirect to the player entity. Would probably be cleaner even now, but I'll leave that up to you.